mmdet3d/core/visualizer/show_result.py

```diff
@@ -87,10 +90,8 @@ def show_result(points, gt_bboxes, pred_bboxes, out_dir, filename, show=True):
             vis.add_bboxes(bbox3d=pred_bboxes)
         if gt_bboxes is not None:
             vis.add_bboxes(bbox3d=gt_bboxes, bbox_color=(0, 0, 1))
-        vis.show()
-
-    result_path = osp.join(out_dir, filename)
-    mmcv.mkdir_or_exist(result_path)
+        show_path = osp.join(result_path, f'{filename}_online.png')
+        vis.show(show_path)
```

Comment:
I don't think we should save the online results every time, at least let's pass a parameter to determine whether to save? Like what I've done [here](https://github.com/Wuziyi616/mmdetection3d/blob/multi_dataset_browse/mmdet3d/core/visualizer/show_result.py#L102-105)?

Reply:
Thanks for your advice. I have also considered the problem of mandatory saving online results. I will modify the corresponding operation lately.